net/sslsplit/Makefile

```diff
@@ -0,0 +1,46 @@
+
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sslsplit
+PKG_VERSION:=0.5.2
```

Comment:
0.5.4 is out

Reply:
0.5.4 has some dependency changes. I'm working on it.